stack.yaml

```diff
@@ -170,7 +170,7 @@ packages:
 # buggy async
 - location:
     git: https://github.com/input-output-hk/log-warper.git
-    commit: 4af8114fbcd5698a4e1fbc653031f0300e29faee
+    commit: 0b7d4d48310f139d75829d31883aaa87ce53312c
```

Comment:
hmm, we're not using `log-warper` anymore I thought.

Reply:
I'll leave https://github.com/input-output-hk/cardano-crypto/pull/46 just here. But add it to the stack.yaml as well.

Reply:
Still used by
- cardano-report-server-0.5.10 (due to cardano-sl-core-2.0.0)
- cardano-sl-faucet-2.0.0
- cardano-sl-utxo

Reply:
Those should probably be fixed to use the new logging method.

Don't worry about it for this PR. Maybe someone on @CodiePP 's team can take care of it.

Reply:
yes, good point. we will investigate ..